albam/blender_ui/export_panel.py

```diff
@@ -261,17 +262,94 @@ def get_selected_item(context):
 
 @blender_registry.register_blender_type
 class ALBAM_OT_Pack(bpy.types.Operator):
+    FILEPATH = bpy.props.StringProperty(
+        name="File Path",
+        description="Filepath used for exporting the file",
+        maxlen=1024,
+        subtype='FILE_PATH',
+    )
+
     bl_idname = "albam.pack"
     bl_label = "Pack item"
+    filepath: FILEPATH
+
+    def invoke(self, context, event):  # pragma: no cover
+        vfs = context.scene.albam.file_explorer
+        index = vfs.file_list_selected_index
+        item = vfs.file_list[index]
+        parent_node = ""
+        if item .is_archive:
+            parent_node = item.display_name
+        else:
+            parent_node = (item.tree_node_ancestors[0].node_id).split("::")[1]
+        self.filepath = parent_node
+        context.window_manager.fileselect_add(self)
+        return {'RUNNING_MODAL'}
 
     def execute(self, context):  # pragma: no cover
-        # TODO
+        vfs_i = context.scene.albam.file_explorer
+        vfs_e = context.scene.albam.exported
+        index = vfs_i.file_list_selected_index
+        item = vfs_i.file_list[index]
+        parent_node = ""
+        #print("selected item name {}".format(item.name))
```

Comment:
Please run `flake8` or other linting tool and fix the issues it reports. Here it would tell you that comments need a space , but I'd prefer to remove unused code altogether.
I'm trying to keep the code base clean, and will add automatic checks in the future.